install_ffmpeg.sh

```diff
@@ -15,24 +42,34 @@ if [ ! -e "$HOME/nasm/nasm" ]; then
   make install || echo "Installing docs fails but should be OK otherwise"
 fi
 
-if [ ! -e "$HOME/x264/x264" ]; then
+if [ ! -e "$HOME/x264" ]; then
```

Comment:
I'm OK with this. The old way gave us a fighting chance of restarting successfully if the build failed for whatever reason (or had cloned but not built), but the behavior wasn't that reliable anyway. If we wanted to be consistent with the ffmpeg check, we could check for `$HOME/x264/libx264.a`

Reply:
Yeah - rather than trying to make this script fully idempotent I'd be more inclined to just cache the Mac version of the `~/compiled` folder after it's produced in the CI process and allow devs to download it.

Reply:
Kinda funny how we're halfway to inventing autotools and packagers now

Reply:
Don't tempt me or `go-livepeer` will have a `package.json` file before you know it 😃